ros_ws/src/crazyswarm/scripts/test_collisionAvoidance.py

```diff
@@ -1,37 +1,39 @@
 import sys
-
 import numpy as np
-
```

Comment:
These blank lines are standard python style: https://pep8.org/#imports

The idea is to group the imports into

```
standard library

third-party libraries

our own code
```